src/dso_api/dynamic_api/routers.py

```diff
@@ -50,58 +48,40 @@
     from schematools.contrib.django.models import DynamicModel
 
 
-class DynamicAPIRootView(APIView):
+class DynamicAPIRootView(APIIndexView):
     """An overview of API endpoints."""
 
     name = "DSO-API"  # for browsable API.
     description = (
         "To use the DSO-API, see the documentation at <https://api.data.amsterdam.nl/v1/docs/>. "
     )
-    schema = None  # exclude from schema
-    dataset_ids: List[str] = []  # set by as_view()
-
-    def get(self, request, *args, **kwargs):
-        base = request.build_absolute_uri("/").rstrip("/")
-        datasets = list(Dataset.objects.filter(id__in=self.dataset_ids))
-        result = {"datasets": {}}
-
-        for ds in datasets:
-            dataset_id = ds.schema.id
-            result["datasets"][ds.schema.id] = {
-                "id": ds.schema.id,
-                "name": ds.name,
-                "title": ds.schema.title or "",
-                "status": ds.schema.get("status", "Beschikbaar"),
-                "description": ds.schema.description or "",
-                "api_type": "rest_json",
-                "api_url": base + reverse(f"dynamic_api:openapi-{dataset_id}"),
-                "documentation_url": f"{base}/v1/docs/datasets/{dataset_id}.html",
-                "specification_url": base + reverse(f"dynamic_api:openapi-{dataset_id}"),
-                "terms_of_use": {
-                    "government_only": "auth" in ds.schema,
-                    "pay_per_use": False,
-                    "license": ds.schema.license,
-                },
-                "related_apis": [],
+    api_type = "rest_json"
+
+    def get_environments(self, ds: Dataset, base: str):
```

Comment:
This could use a return type. (eg `-> List[Dict[str, str]]`) but see also next comment.